### PR TITLE
Add content on token-suppliers dashboard

### PR DESCRIPTION
**Pull request by burieberry**
token-balances-depot

### CODE REVIEW

**Comment by lukemelia** [comment]
@Aierie can you rebase and update this PR to fit the new bridged token symbol approach?

**Review by lukemelia — COMMENTED** [review]
Looks good, just needs to be rebased and updated to use bridge token symbols from SDK

**Comment by Aierie** [comment]
> @Aierie can you rebase and update this PR to fit the new bridged token symbol approach?

Sure

File: packages/web-client/app/services/token-to-usd.ts [hunk 1]
```diff
@@ -88,7 +88,18 @@ export default class TokenToUsd extends Service {
...
+    amount: BN
+  ): number | undefined {
+    if (symbol === 'DAI') {
+      symbol = 'DAI.CPXD';
+    }
+
+    if (symbol === 'CARD') {
+      symbol = 'CARD.CPXD';
+    }
+
```
Comment (Aierie) [inline packages/web-client/app/services/token-to-usd.ts:102]: The way we're doing the usd conversion is a bit convoluted right now. We accept `.CPXD` here, but in `layer2-chain` we remove it. I'll clean it in a followup PR.

**Review by lukemelia — APPROVED** [review]

File: packages/web-client/app/services/token-to-usd.ts (hunk 1 above)
Comment (burieberry) [reply packages/web-client/app/services/token-to-usd.ts:102]: Thanks! merging this

**Pull request merged**